dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -230,6 +230,10 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
     CUDA_CALL(cudaStreamCreateWithPriority(
       &hw_decode_stream_, cudaStreamNonBlocking, default_cuda_stream_priority_));
 
+    if (hw_decoder_images_staging_.is_pinned())
```

Comment:
I guess we can move it to the constructor but it should not make any difference after all.

Reply:
NVM